content/en/blog/2023/dlb-connection-balance/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: "Using Accelerated Offload Connection Load Balance in Istio"
```

Comment:
I don't think balance is correct throughout this document. I think it should maybe be balancing or balancer depending on the part of speech.
Example:
```suggestion
title: "Using Accelerated Offload Connection Load Balancing in Istio"
```

Reply:
Updated.